internal/storage/storage.go

```diff
@@ -116,7 +128,7 @@ func NewFactory(cl *ConfigList) (Factory, error) {
 			if cfg.Etcd.ForceClient != nil {
 				cli = cfg.Etcd.ForceClient
 			} else {
-				cli, _ = etcd.NewClient(cfg.Etcd)
+				cli, err = etcd.NewClient(cfg.Etcd)
```

Comment:
Is there any reason why we continue the process if it failed? Both calls `NewFactory` function ends with `fatalOnError` if error shows up.